touch-detect/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, CMake
```

Comment:
Copyright header notice on most of the new files in `touch-detect` are missing

Reply:
Yeah, it was because a bunch of them were auto-generated by conan but you're right, I'll add them.

Reply:
I think at a project level the structure is quite important. My suggestion is to use `conf/` as a bucket for everything configuration. How you organise it lower than that probably varies on what you have, where it's installed, etc